client/my-sites/domains/domain-management/dns/dns-add-new.jsx

```diff
@@ -53,6 +54,18 @@ class DnsAddNew extends React.Component {
 					data: '',
 				},
 			},
+			{
+				component: AliasRecord,
+				types: [ 'ALIAS' ],
+				description: translate(
+					'ALIAS (domain alias) records are typically used to direct one domain (e.g. example.com) to another domain (e.g. example.net).'
```

Comment:
I wonder if we should mention something in terms of "ALIAS is a non-standard DNS record and the target domain is resolved on the DNS server"?